javascripts/discourse/widgets/layouts-groups.js.es6

```diff
@@ -20,7 +20,7 @@ export default layouts.createLayoutsWidget('group-list', {
         'a.layouts-group-list-header',
         {
           attributes: {
-            href: '/g?type=my',
+            href: settings.header_link,
```

Comment:
add conditional:
```js
href: settings.header_link || '/g'
```

Reply:
Thanks - but that isn't quite what I meant. This will mean that it fallsback to /g if the setting is `null`

I think that if the setting has been made `null` then there should be no link at all.

Reply:
@nathan-nz Ah I see,

In that case you can probably do something like:

```js
href: settings.header_link || ‘#’

```

Reply:
I've just tried that, but it does weird stuff.

I guess what I'm trying to achieve is having it as a 'dead' header which is just text, with no link. But really, this isn't worth chasing as when the setting is left blank now it just links to the homepage - which is fine. 

I think that you can safely commit it.

Reply:
@nathan-nz Hmm, I thought that might be a simple quick approach, but we can always do it explicitly with a conditional. Something like this should work:
```js
      if (settings.header_link) {
        return h(
          'a.layouts-group-list-header',
          {
            attributes: {
              href: settings.header_link,
              title: I18n.t(themePrefix('groups_widget.title')),
            },
          },
          I18n.t(themePrefix('groups_widget.title'))
        );
      } else {
        return h('h3.layouts-group-list-header', I18n.t(themePrefix('groups_widget.title')));
      }
```

If you'd like you can give this a try and test to see if it works. Otherwise, i can take a look at testing it when I have more time.